### DISCUSSION TITLE
Automatic Skyblock Name

### DISCUSSION THREAD

**Discussion by MatthewXcap** [BG-Software-LLC/SuperiorSkyblock2#1737] [Pending Feature Requests]
It would be nice if whenever new players create an island, if they do not add an island name, it doesnt tell them to retype the command but instead creates the island name as "Username's Island". 

Also I'd like to add that I understand that you're able to disable island names in config, but I think they should be able to choose whether or not they want to name their island. The only issue that would arise from this is that players would not know that they can give their island a name. But eventually they'd realize somehow if they play for long enough and use /is rename.

**Comment by OmerBenGera** [comment]
What happens if someone creates an island with the name of "X's Island", then a player named "X" creates an island? There will be a conflict.
I don't like the idea of forcing islands to have specific names, especially if they turn to be meaningless, such as in your scenario.

**Reply by MatthewXcap** [reply]
I agree with this. I think maybe it could be a config thing set by the server owner. Like "Use custom island names". So it would use the variable of the players name with "'s Island" and no real way to set an island name. Maybe server owner's don't want islands to be given profane names and don't want to have to deal with monitoring that.
 

However, if custom names are enabled.. Which they should be by default in this scenario, it would remain exactly how it is, forcing the player to choose an island name. 

I also understand players want to customize the name of things.. They could always use a warp name. Which is already a feature.
